charts/nginx-gateway-fabric/templates/clusterrole.yaml

```diff
@@ -8,13 +8,15 @@ rules:
 - apiGroups:
   - ""
   - apps
+  - autoscaling
```

Comment:
Do we not want these to be conditional based on if autoscaling is enabled?

Reply:
Originally that's how we had it, but we have to define the controller to actually watch these resources, and that is at initialization, and I don't believe we can make it dynamic based on whether a user toggles on or off. So we always have to be able to watch these.